gallery_dl/extractor/furaffinity.py

```diff
@@ -132,11 +135,16 @@ def _parse_post(self, post_id):
             data["_description"] = extr(
                 '<td valign="top" align="left" width="70%" class="alt1" '
                 'style="padding:8px">', '                               </td>')
+            data["folders"] = []  # folders not present in old layout
```

Comment:
```suggestion
            data["folders"] = ()  # folders not present in old layout
            
```

Empty tuple is more lightweight than constructing a new empty list every time.